owncloudApp/src/main/java/com/owncloud/android/presentation/ui/sharing/fragments/ShareFileFragment.kt

```diff
@@ -155,11 +153,10 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
             if (!isDefaultNameSet) {
                 return defaultName
             }
-            Collections.sort(usedNumbers)
-            var chosenNumber = -1
-            if (usedNumbers.size == 0 || usedNumbers[0] != 2) {
-                chosenNumber = 2
-
+            usedNumbers.sort()
+            var chosenNumber = UNUSED_NUMBER
+            if (usedNumbers.size == 0 || usedNumbers[0] != USED_NUMBER_SECOND) {
```

Comment:
you can refactor this if statement with 

`if (usedNumbers.firstOrNull() != USED_NUMBER_SECOND)`

Reply:
Done